Drukhari/Readme.md

```diff
@@ -0,0 +1,96 @@
+  * **`Sslyth:`** <br> Enable the profile ability `-1 to wound roll (if defender)`.
+
+* #### Court of the Archon + Archon:
+  * **`Overlord:`** *Leader Ability* <br> Enable the profile ability `Re-roll all wound roll results of 1 (if attacker)`.
+  * **`Overlord (Empowered):`** *Leader Ability* <br> Enable the profile ability `Re-roll all possible failed wound rolls (if attacker)`.
+  * **`Lhamaean:`** <br> Enable the profile ability `LETHAL HITS`.
+  * **`Sslyth:`** <br> Enable the profile ability `-1 to wound roll (if defender)`.
+
+* #### Drazhar:
+  * **`Master of Blades:`** *Leader Ability* <br> Enable the profile ability `+1 to wound roll (if attacker)`.
+
+* #### Haemonculus:
+  * **`Master of Pain:`** *Leader Ability* <br> Enable the profile ability `Feel no pain 4+`.
+
+* #### Kabalite Warriors x10 + Archon:
+  * **`Overlord:`** *Leader Ability* <br> Enable the profile ability `Re-roll all wound roll results of 1 (if attacker)`.
+  * **`Overlord (Empowered):`** *Leader Ability* <br> Enable the profile ability `Re-roll all possible failed wound rolls (if attacker)`.
+
+* #### Incubi x5 + Drazhar:
+  * **`Master of Blades:`** *Leader Ability* <br> Enable the profile ability `+1 to wound roll (if attacker)`.
+
+* #### Lelith Hesperax:
+  * **`Brides of Death:`** *Leader Ability* <br> Enable the profile ability `+1 to strength (melee only) (if attacker)`.
+  * **`Brides of Death:`** *Leader Ability* <br> Enable the profile ability `Improve AP by 1 (melee only) (if attacker)`.
+  * **`Thrilling Spectacle:`** <br> Enable the profile ability `3++ invuln save`.
+  * **`Thrilling Spectacle:`** <br> Use the weapon profile `Lelith's blades (Thrilling Spectacle)`.
+
+* #### Ravager:
+  * **`Eradicate the Foe:`** <br> Enable the profile ability `Re-roll all hit roll results of 1 (if attacker)`.
+
+* #### Razorwing Jetfighter:
+  * **`Ground Attack Craft:`** <br> Enable the profile ability `+1 to hit roll (if attacker) (excludes FLY)`.
+
+* #### Reavers:
+  * **`Grav-talon:`** <br> Enable the profile ability `+1 to wound roll (if attacker charged) (melee only) (if attacker)`.
+
+* #### Succubus:
+  * **`Storm of Blades:`** *Leader Ability* <br> Enable the profile ability `SUSTAINED HITS 1`.
+
+* #### Wracks x5 + Haemonculus:
+  * **`Master of Pain:`** *Leader Ability* <br> Enable the profile ability `Feel no pain 4+`.
+
+* #### Wyches x10 + Lelith Hesperax:
+  * **`Brides of Death:`** *Leader Ability* <br> Enable the profile ability `+1 to strength (melee only) (if attacker)`.
+  * **`Brides of Death:`** *Leader Ability* <br> Enable the profile ability `Improve AP by 1 (melee only) (if attacker)`.
+  * **`Thrilling Spectacle:`** <br> Not used, as attacks are allocated to bodyguard units first.
+  * **`Thrilling Spectacle:`** <br> Use the weapon profile `Lelith's blades (Thrilling Spectacle)`.
+
+* #### Wyches x10 + Succubus:
+  * **`Storm of Blades:`** *Leader Ability* <br> Enable the profile ability `SUSTAINED HITS 1`.
```

Comment:
### Additional Changes:

Hellions:
* Remove `Infantry` keyword.

Talos x1, Talos x2:
* `Twin Drukhari haywire blaster` weapon, change Damage to 3.

Voidraven Bomber:
* Missing `Fly` keyword.

Wyches x10:
* `Splinter pistol` Weapon: `Anti-infantry 4+` ability needs to be changed to `Anti-infantry 3+`.
* Add the Profile ability `4++ invuln save (melee only)`